tests/try_assert.py

```diff
@@ -0,0 +1,43 @@
+"""
+This test is executed separately due to pytest's assertion-rewriting
```

Comment:
I think you suggested naming this file `assert_test.py`, but that still results in pytest trying to run it. So I changed `test` to `try` to prevent that. Let me know if you'd prefer a different name.

Reply:
Sounds good to me.